### DISCUSSION TITLE
Does Traces Panel can only used with Jaeger's TraceID search type?

### DISCUSSION THREAD

**Discussion by JobberRT** [grafana/grafana#69687] [Dashboards, Panels & Visualizations]
Using other types of search will causing this error:
<img width="800" alt="image" src="https://github.com/grafana/grafana/assets/17498767/dbdacecc-c9a0-404d-a783-d7e6504a8f19">

I would like to achieve a panel just like Jaeger does:
- a breif dot graph showing duration
<img width="925" alt="image" src="https://github.com/grafana/grafana/assets/17498767/f526add8-425c-4734-bbb1-54fc1aca54d8">
- a detail graph showing sub-span duration or detail info
<img width="1246" alt="image" src="https://github.com/grafana/grafana/assets/17498767/0ef9741c-50c7-4e59-99c9-e734ad25f0ba">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82641. Feel free to continue the discussion around this there. Thank you!
